Doc/library/functions.rst

```diff
@@ -747,6 +747,14 @@ are always available.  They are listed here in alphabetical order.
 
    .. versionchanged:: 3.6
       Grouping digits with underscores as in code literals is allowed.
+ 
+   .. versionchanged:: 3.6.16
```

Comment:
3.6.16 will never exist. For Fedora purposes, we can use version-release, like this:

```suggestion
   .. versionchanged:: 3.6.15-12
```

Reply:
Oh right, I replaced version with ``3.6.15-13``.